static/js/components/notifications/index.js

```diff
@@ -38,7 +42,26 @@ export const UnusedCouponNotification = (
   )
 }
 
+export const B2BOrderStatusNotification = (props: ComponentProps) => {
+  const { dismiss } = props
+
+  return (
+    <span>
+      Something went wrong. Please contact us at{" "}
+      <a
+        href="https://xpro.zendesk.com/hc/en-us/requests/new"
+        onClick={dismiss}
+        className="alert-link"
+      >
+        Customer Support
+      </a>
+      .
+    </span>
+  )
+}
+
```

Comment:
Looks good. Can we add a test for this component?

Reply:
sure